doc/man/meson.build

```diff
@@ -22,8 +22,13 @@ man5 = [
     'bluechi-agent.conf',
 ]
 
+man8 = [
+    'bluechi-controller.selinux',
+    'bluechi-agent.selinux'
```

Comment:
NIT:
I'd exchange the dot with a dash here so you get the installed man page like this
```bash
man bluechi-controller-selinux
```

```suggestion
    'bluechi-controller-selinux',
    'bluechi-agent-selinux'
```

Reply:
Changed)

Reply:
Please also rename `doc/man/bluechi-agent.selinux.8.md` to `doc/man/bluechi-agent-selinux.8.md` (the same for the controller). This is causing the build to fail at the moment.

Reply:
@schastlivcev Could you do the renaming? I think then this PR is ready to merge. 
We intend to release v0.8 next week and it would be great to have this PR in there :)

Reply:
Done

Reply:
Thanks! Builds are passing.